src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderAssessmentResourceTest.java

```diff
@@ -192,6 +192,47 @@ public void testGetOffenderCategorisationsSystem() {
         assertThatJson(response.getBody()).node("[5].bookingId").isEqualTo(value(-41));
     }
 
+    @Test
```

Comment:
Be good to have a test to check for regressions around permissions.

Reply:
done